packages/plugin-client-ip/src/client-ip.ts

```diff
@@ -0,0 +1,37 @@
+import { Client, Plugin } from 'packages/core/types'
+import assign from '@bugsnag/core/lib/es-utils/assign'
+
+interface InternalClient extends Client {
+  _config: {
+    collectUserIp: boolean
+  }
+}
+
+/*
+ * Prevent collection of user IPs
+ */
+const plugin: Plugin = {
```

Comment:
you can also change this to 

```interface ExtendedPlugin extends Plugin {
  configSchema: Record<string, ValidationOption>
}```

to remove the need for the second `@ts-expect-error`